packages/@coorpacademy-components/src/molecule/bulk-empty-dashboard/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonLink from '../../atom/button-link';
+import style from './style.css';
+
+const BulkEmptyDashboard = ({mainText, subText, imageUrl, buttonLink}) => {
+  return (
+    <div className={style.container}>
+      <div>
+        <img className={style.img} src={imageUrl} />
```

Comment:
l'image n'est pas porteuse d'information (que pour la déco)
```suggestion
        <img className={style.img} src={imageUrl} aria-hidden="true" />
```